src/tiledb/cloud/utilities/_common.py

```diff
@@ -382,9 +382,20 @@ def as_batch(func: _CT) -> _CT:
     - acn: Access Credentials Name (ACN) registered in TileDB Cloud (ARN type)
     - resources: resources to allocate for the UDF, defaults to None
 
-    :param func: function to run
+    :param func: function to run or name of registered UDF
     """
 
+    if isinstance(func, str):
+        remote_udf = True
+        registered_udf = func
+
+        def func():
```

Comment:
Is there a better way? Need to get a function that we can wrap, so I just make a dummy one and then overwrite its name attr...